tdnf.spec.in

```diff
@@ -150,80 +157,61 @@ cmake \
 
 %make_build
 %make_build python
-
-%if 0%{?with_check}
```

Comment:
Why is this changed to unconditional?

Reply:
Yeah, better to keep it in this case. Thanks.